src/main/resources/admin/widgets/contentviewer/contentviewer.js

```diff
@@ -64,7 +60,7 @@ function highlightJson(json) {
     json = json.replace(/&/g, '&amp;').replace(/</g, '&lt;').replace(/>/g, '&gt;');
     return json.replace(/("(\\u[a-zA-Z0-9]{4}|\\[^u]|[^\\"])*"(\s*:)?|\b(true|false|null)\b|-?\d+(?:\.\d*)?(?:[eE][+\-]?\d+)?)/g,
         function (match) {
-            var cls = 'number', title;
+            let cls = 'number', title;
```

Comment:
We don't really use this syntax for initiating variables.
Can we define two variables instead of combining them?